bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/ReplicationWorker.java

```diff
@@ -330,14 +330,14 @@ private boolean rereplicate(long ledgerIdToReplicate) throws InterruptedExceptio
                 return false;
             }
 
-        } catch (BKNoSuchLedgerExistsException e) {
+        } catch (BKNoSuchLedgerExistsOnMetadataServerException e) {
             // Ledger might have been deleted by user
-            LOG.info("BKNoSuchLedgerExistsException while opening "
+            LOG.info("BKNoSuchLedgerExistsOnMetadataServerException while opening "
                 + "ledger {} for replication. Other clients "
                 + "might have deleted the ledger. "
                 + "So, no harm to continue", ledgerIdToReplicate);
             underreplicationManager.markLedgerReplicated(ledgerIdToReplicate);
-            getExceptionCounter("BKNoSuchLedgerExistsException").inc();
+            getExceptionCounter("BKNoSuchLedgerExistsOnMetadataServerException").inc();
```

Comment:
its not ideal to use hardcoded values, better to create static variable and use it here.

Reply:
Agree; I just changed the exception type in this checkin.